### DISCUSSION TITLE
Can I change the character to as lawyer, or doctor when the chatbot replies to my messages?

### DISCUSSION THREAD

**Discussion by jonnytri53** [mayooear/ai-pdf-chatbot-langchain#348] [Q&A]
Hi, community,
Can I change the character and make the chatbot reply as a lawyer, doctor, or mechanic engineering,.... other than as an AI language model. How can I achieve it? Is there any folder for me to change it? Thank you so much, everyone. I'd appreciate it.

**Comment by Maxton-Huff** [comment]
Try changing the prompt in utils\makechain.ts to whatever character you are looking for.
